modules/navier_stokes/test/tests/finite_volume/ins/natural_convection/linear_segregated/2d/diff_heated_cavity_linear_segregated.i

```diff
@@ -6,15 +6,15 @@ T_0 = 875.0
 mu = 1.
 k_cond = 38.0
 cp = 640.
-alpha = 3.26e-4
+alpha = 3.26e-5
```

Comment:
It feels like these two input files in this directory test the same things now, should we just keep one of them?

Reply:
Okay never mind, these are different. One is just boussinesq. Resolving this guy.